migrate/20250131_drop_hetzner_host.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    drop_table :hetzner_host
+  end
+
+  down do
+    alter_table(:host_provider) do
+      drop_column :provider_name
+    end
```

Comment:
The up block does not add a provider_name column to host_provider, so the down block should not drop the column.  Instead of using `LIKE host_provider INCLUDING ALL`, you need to recreate the hetzner host table as it was, and then populate it.